hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/BackpressureRegulator.java

```diff
@@ -6,8 +6,9 @@
 import com.hazelcast.spi.Operation;
 
 import java.util.Random;
-import java.util.concurrent.atomic.AtomicInteger;
 
+import static com.hazelcast.client.impl.protocol.util.BitUtil.CACHE_LINE_LENGTH;
```

Comment:
Guys,
Client Protocol is being implemented as partial PR's and we will make the code clean up including  merging the redundant utils to hazelcast util as a last step of this process so please do not use anything from protocol classes until we finalise them.